sunpy/visualization/colormaps/color_tables.py

```diff
@@ -57,15 +58,15 @@ def paden(lst, len_, pad=0):
         205, 207, 208, 210, 211, 212, 214, 215, 217, 218, 220, 221, 223, 224,
         226, 227, 228, 230, 231, 233, 234, 236, 237, 239, 240, 241, 243, 244,
         246, 247, 249, 250, 252, 253
-    ], 256, 255))
+    ], 256, 255))*u.angstrom
 
-g0 = np.array(padfr(_mkx(1, range(17, 256, 17), 2), 256))
-b0 = np.array(padfr(_mkx(3, range(51, 256, 51), 4), 256))
+g0 = np.array(padfr(_mkx(1, range(17, 256, 17), 2), 256))*u.angstrom
+b0 = np.array(padfr(_mkx(3, range(51, 256, 51), 4), 256))*u.angstrom
 
-c0 = np.arange(256, dtype='f')
-c1 = (np.sqrt(c0) * np.sqrt(255.0)).astype('f')
-c2 = (np.arange(256)**2 / 255.0).astype('f')
-c3 = ((c1 + c2 / 2.0) * 255.0 / (c1.max() + c2.max() / 2.0)).astype('f')
+c0 = np.arange(256, dtype='f')*u.angstrom
+c1 = (np.sqrt(c0) * np.sqrt(255.0)).astype('f')*u.angstrom
+c2 = (np.arange(256)**2 / 255.0).astype('f')*u.angstrom
+c3 = ((c1 + c2 / 2.0) * 255.0 / (c1.max() + c2.max() / 2.0)).astype('f')*u.angstrom
```

Comment:
You don't want to change the table data to be units. These are used by matplotlib to color in images.

Reply:
yes..
I'm sorry.. i failed ,any hints please!

Reply:
That's fine! This is all new to you, we can take this slowly.

Reply:
okay thank you so much...